Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -544,6 +544,19 @@ Virus Total provides an API for analyzing URLs and local files for known threats
 
 ### Static Analysis
 
+As we mentioned before, [handling page navigation](https://developer.android.com/guide/webapps/webview#HandlingNavigation "Handling page navigation") should be analyzed carefully, especially when you suspect that the user might be able to open any external pages. The default and the safest behaviour on Android is to the default web browser open any link that the user might click. However, the developers might decide not to do this and handle the links within their own WebView by providing a `WebViewClient`. If this is the case, be sure to search for and inspect the following interception callback functions:
+
+- `shouldOverrideUrlLoading` allows your application to either abort loading WebViews with suspicious content by returning `true` or allow the WebView to load the URL by returning `false`. Considerations:
+  - This method is not called for POST requests.
+  - This method is not called for XmlHttpRequests, iFrames, "src" attributes included in HTML or `<script>` tags. Instead, `shouldInterceptRequest` should take care of this.
+- `shouldInterceptRequest` allows the application to return the data from resource requests. If the return value is null, the WebView will continue to load the resource as usual. Otherwise, the return response and data will be used. Considerations:
```

Comment:
```suggestion
- `shouldInterceptRequest` allows the application to return the data from resource requests. If the return value is null, the WebView will continue to load the resource as usual. Otherwise, the data returned by the shouldInterceptRequest method is used. Considerations:
```